modules/core/src/main/java/org/apache/ignite/spi/communication/tcp/AttributeNames.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.spi.communication.tcp;
+
+import org.apache.ignite.cluster.ClusterNode;
+
+/**
+ * This class was created for the refactoring approach. It contains attribute names into a {@link ClusterNode}.
+ * It should be remove after global refactoring. The values of these attributes should pass via constructors of appropriate classes.
+ */
+public class AttributeNames {
```

Comment:
Please, mark as 'Depreacated'.
This class seems weird, a enum can be used instead.
What is blocks you from removing this class right now? 

If there is no chance to remove it now, please, create a ticket for this please.

Reply:
https://ggsystems.atlassian.net/browse/GG-29546